contracts/tokenbridge/ethereum/gateway/L1OrbitERC20Gateway.sol

```diff
@@ -25,6 +25,9 @@ contract L1OrbitERC20Gateway is L1ERC20Gateway {
         uint256 _gasPriceBid,
         bytes calldata _data
     ) public payable override returns (bytes memory res) {
+        // fees are paid in native token, so there is no use for ether
+        require(msg.value == 0, "NO_VALUE");
```

Comment:
this seems clearer as a revert reason?
```suggestion
        require(msg.value == 0, "NONZERO_VALUE");
```

Reply:
Right, it can be clearer, but wanted to have it consistent with existing value check here https://github.com/OffchainLabs/token-bridge-contracts/blob/5052476f6ac63f35117cb0b4df1c5573d8a1934d/contracts/tokenbridge/arbitrum/gateway/L2ArbitrumGateway.sol#L147